spec/features/idv/in_person_spec.rb

```diff
@@ -109,7 +118,28 @@
       expect(page).to have_content(t('idv.titles.come_back_later'))
       expect(page).to have_current_path(idv_come_back_later_path)
 
-      # WILLFIX: After LG-6897, assert that "Ready to Verify" content is shown after code entry.
+      click_idv_continue
+      expect(page).to have_current_path(account_path)
+      expect(page).not_to have_content(t('headings.account.verified_account'))
+      click_on t('account.index.verification.reactivate_button')
+      click_button t('forms.verify_profile.submit')
+
+      expect(page).to have_current_path(idv_in_person_ready_to_verify_path)
+      expect(page).not_to have_content(t('account.index.verification.success'))
+    end
+
+    it 'lets the user clear and start over from gpo confirmation', allow_browser_log: true do
```

Comment:
What is `allow_browser_log`?

Reply:
> What is `allow_browser_log`?

By default, a feature spec will automatically fail if there is any console logging that happens, which is meant as a way to avoid accidental `console.log` or any unaccounted JavaScript errors.

Unfortunately, based on the way that unsuccessful doc auth is handled, it also emits an error log. Since we expect that in-person proofing will require an unsuccessful doc auth, we also need to allow for browser logging.

Out-of-scope, but I've been wondering if there might be a more elegant way to handle this though, something like:

1. Changing the boolean value to a string or regexp, so that it's clearer what logging we're actually expecting
2. Exempting the unsuccessful result in a way [similar to what we're doing already for polling](https://github.com/18F/identity-idp/blob/b25790b34d3c542f00849808dc37e8d0a7ec64e7/app/controllers/idv/personal_key_controller.rb#L36-L37)
3. Refactor the doc auth screen to avoid logging errors